src/externals/mct/mct/m_GlobalSegMap.F90

```diff
@@ -1764,22 +1761,25 @@ subroutine active_pes_(GSMap, n_active, pe_list)
   character(len=*),parameter :: myname_=myname//'::active_pes_'
 
   integer :: count, i, n, ngseg, ierr
-  logical :: new
-  integer, dimension(:), allocatable :: temp_list
-  integer, dimension(:), allocatable :: perm
+  integer :: max_activepe, p
```

Comment:
git blame says you did in commit f4a104b1 in PR #1568 .

Its all white space except for changing the last line from "end" to "end program test".

@jgfouca you can take the ESMCI version.  Its better and I'll make the same change in mpi-serial.